inyoka_theme_default/templates/portal/privmsg/base.html

```diff
@@ -0,0 +1,24 @@
+{#
+  portal/privmsg/base.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% from 'macros.html' import breadcrumb_item, sidebar, sidebar_item %}
+{%- extends 'portal/base.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+{% endblock %}
+
+{% block sidebar %}
+  {% call macros.sidebar(_('Messages')) %}

```

Comment:
I would remove 'Messages' from the sidebar and instead append something to the breadcrumb (as mentioned)